src/main/java/edu/gatech/cs2340/trydent/internal/KeyboardImpl.java

```diff
@@ -18,6 +21,235 @@
+        swingToJavaFXKeyMap.put(KeyEvent.VK_Q, KeyCode.Q);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_QUOTE, KeyCode.QUOTE);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_QUOTEDBL, KeyCode.QUOTEDBL);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_R, KeyCode.R);
+        // swingToJavaFXKeyMap.put(, KeyCode.RECORD);
+        // swingToJavaFXKeyMap.put(, KeyCode.REWIND);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_RIGHT, KeyCode.RIGHT);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_RIGHT_PARENTHESIS, KeyCode.RIGHT_PARENTHESIS);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_ROMAN_CHARACTERS, KeyCode.ROMAN_CHARACTERS);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_S, KeyCode.S);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_SCROLL_LOCK, KeyCode.SCROLL_LOCK);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_SEMICOLON, KeyCode.SEMICOLON);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_SEPARATOR, KeyCode.SEPARATOR);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_SHIFT, KeyCode.SHIFT);
+        // swingToJavaFXKeyMap.put(, KeyCode.SHORTCUT);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_SLASH, KeyCode.SLASH);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_0);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_1);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_3);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_4);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_5);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_6);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_7);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_8);
+        // swingToJavaFXKeyMap.put(, KeyCode.SOFTKEY_9);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_SPACE, KeyCode.SPACE);
+        // swingToJavaFXKeyMap.put(, KeyCode.STAR);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_STOP, KeyCode.STOP);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_SUBTRACT, KeyCode.SUBTRACT);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_T, KeyCode.T);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_TAB, KeyCode.TAB);
+        // swingToJavaFXKeyMap.put(, KeyCode.TRACK_NEXT);
+        // swingToJavaFXKeyMap.put(, KeyCode.TRACK_PREV);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_U, KeyCode.U);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_UNDEFINED, KeyCode.UNDEFINED);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_UNDERSCORE, KeyCode.UNDERSCORE);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_UNDO, KeyCode.UNDO);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_UP, KeyCode.UP);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_V, KeyCode.V);
+        // swingToJavaFXKeyMap.put(, KeyCode.VOLUME_DOWN);
+        // swingToJavaFXKeyMap.put(, KeyCode.VOLUME_UP);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_W, KeyCode.W);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_WINDOWS, KeyCode.WINDOWS);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_X, KeyCode.X);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_Y, KeyCode.Y);
+        swingToJavaFXKeyMap.put(KeyEvent.VK_Z, KeyCode.Z);
+    }
```

Comment:
This is a horrifying table but looks like it get the job done. I suggest updating it in a future PR to use java reflection to copy the fields over, since it looks there you can just find static fields that match the pattern "VK_*". I can do this in a follow-up PR if you like, I'm OK with merging it now.